projects/driver_controls/test/test_can_output.c

```diff
@@ -88,17 +84,34 @@ void teardown_test(void) { }
 
 // Test that the power fsm correctly generates CAN events
 void test_can_output_power(void) {
-  prv_toggle_power(true);
+  TEST_ASSERT_EQUAL_STRING("state_off", s_fsm_group.power.current_state->name);
+
+  prv_toggle_power();
```

Comment:
I'd like some more rigorous testing to ensure that we don't enter charge mode accidentally. Also, that we don't try to drive the motors in the charge mode.